lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_publication
+short_description: Add or remove PostgreSQL publication from a remote host
+description:
+   - Add or remove PostgreSQL publication from a remote host.
+version_added: "2.5"
+options:
+  name:
+    description:
+      - Name of the publication to add or remove.
+    required: true
+    default: null
+  database:
+    description:
+      - Name of the database to connect to.
+    required: false
+    default: postgres
+  tables:
+    description:
+      - List of the tables to target for publication.
```

Comment:
Can you add something about the default targetting all tables (e.g., if not set, then it do for all tables) ? I suspect this might be obvious for someone knowing postgresql, but not everybody know

Reply:
done